packages/ui/src/components/va-data-table/hooks/useSortable.ts

```diff
@@ -14,7 +14,7 @@ export const useSortableProps = {
   ...useThrottleProps,
   sortBy: { type: String as PropType<string | undefined> },
   columnSorted: { type: Object as PropType<any | undefined> },
-  sortingOrder: { type: String as PropType<DataTableSortingOrder | undefined> },
+  sortingOrder: { type: String as PropType<DataTableSortingOrder | null | undefined> },
```

Comment:
The original issue was a runtime warning from vue, not ts error. Looks like `DataTableSortingOrder` already includes `null`.

```suggestion
  sortingOrder: { type: [String, null] as PropType<DataTableSortingOrder | undefined> },
```